packages/hardhat-core/test/internal/core/providers/gas-providers.ts

```diff
@@ -256,6 +256,98 @@ describe("AutomaticGasPriceProvider", () => {
       });
     });
 
+    describe("When the eth_feeHistory result causes maxPriorityFeePerGas to be 0 and eth_maxPriorityFeePerGas doesn't exist", function () {
+      const latestBaseFeeInMockedProvider = 80;
+
+      beforeEach(function () {
+        mockedProvider.setReturnValue("eth_feeHistory", {
+          baseFeePerGas: [
+            numberToRpcQuantity(latestBaseFeeInMockedProvider),
+            numberToRpcQuantity(
+              Math.floor((latestBaseFeeInMockedProvider * 9) / 8)
+            ),
+          ],
+          reward: [["0x0"]],
+        });
+
+        mockedProvider.setReturnValue("eth_getBlockByNumber", {
+          baseFeePerGas: "0x1",
+        });
+      });
+
+      it("should use the reward return value as default maxPriorityFeePerGas", async function () {
```

Comment:
is this description correct? Shouldn't it be the default value 0x1?

Reply:
Good catch, fixed.